test/WebJobs.Script.Tests.Integration/Host/StandbyManagerTests.cs

```diff
@@ -328,10 +329,17 @@ protected async Task InitializeTestHostAsync(string testDirName, IEnvironment en
                     // This source reads from AzureWebJobsScriptRoot, which does not work
                     // with the custom paths that these tests are using.
                     var source = c.Sources.OfType<WebScriptHostConfigurationSource>().SingleOrDefault();
+                    
                     if (source != null)
                     {
                         c.Sources.Remove(source);
                     }
+
+                    var bundleSource = c.Sources.OfType<ExtensionBundleConfigurationSource>().SingleOrDefault();
+                    if (bundleSource != null)
+                    {
+                        c.Sources.Remove(bundleSource);
```

Comment:
Why is this needed?

Reply:
This test requires you to remove sources that rely on script root. When i added a new source it broke this test. I changed it to fix the test and get it back to passing state.